### ISSUE TITLE
Finetuning with CuLane dataset on a model trained with TuSimple

### ISSUE THREAD

**Issue by alecapecchi** [Turoad/lanedet#66]
Hello,
I have tried to fine tune the laneatt_r18_tusimple.pth model, by running this command:

`python main.py configs/laneatt/resnet18_culane.py --finetune_from pretrained/laneatt_r18_tusimple.pth --gpus 1`

But at the end I still get Best metric: 0.0

Any idea of what could I cange to improve it?

**Comment by alecapecchi** [comment]
Ok, I was linking to the wrong dataset. My bad
